<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lslVisual</class>
 <widget class="QMainWindow" name="lslVisual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LSL Visualization</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="queryButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Query</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Available Streams/Markers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="visualizeButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>90</y>
      <width>281</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualize</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="channelView">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>150</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>279</width>
       <height>179</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>130</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Available Channels</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="streamMetaData">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>731</width>
      <height>181</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="UTKlogo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="NBL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nonlinear Biodynamics Lab</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>1061</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="signalViewer"/>
   </widget>
   <widget class="QPushButton" name="stopStream">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>690</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Streaming</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startStream">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>690</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Streaming</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="LSL_Visualization">
    <property name="title">
     <string>LSL Visualization</string>
    </property>
   </widget>
   <addaction name="LSL_Visualization"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
